app/helpers/assignment_index_helper.rb

```diff
@@ -7,17 +7,17 @@ def cal_day(day:)
 
   def cal_assignment(assignment:, day:)
     return if assignment.blank?
-    ass = 'assignment'
-    ass += '-user' if assignment.user == @current_user
+    assign = 'assignment'
+    assign += '-user' if assignment.user == @current_user
 
     if day == assignment.start_date && day == assignment.end_date
```

Comment:
The `start_date` and `end_date` are the same if someone is on call for a single day, e.g. 3pm Friday to 3pm Saturday.